callisto/delivery/hashers.py

```diff
@@ -0,0 +1,231 @@
+import base64
```

Comment:
ah, okay--well, as much as it's a little bit of cutting the baby in half, I'd like to go ahead and subclass the PBKDF2 hasher and we can copy over the Argon2 one. It'll at least provide a good template for subclassing once we update to 1.10 (which should happen soon!) I'll make some bugs around it to track that work separately. Can you add this clause about the license on the Argon2 hasher? https://github.com/SexualHealthInnovations/django-wizard-builder/blob/master/wizard_builder/views.py#L18-L20